public/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
dont put login on this file, it must be render by framework

Reply:
ok